### ISSUE TITLE
Bicep example of backupInstance doesn't configure containersList

### ISSUE THREAD

**Issue by ngbrown** [Azure/azure-quickstart-templates#13570]
This quickstart to backup a storage account is not enough and will not backup any configured containers. Does it need to include `policyParameters.backupDatasourceParametersList[].containersList` inside the `policyInfo`? I can't find any documentation on this.

https://github.com/Azure/azure-quickstart-templates/blob/240b84631ea3eb17252f02ce301f1056219b7870/quickstarts/microsoft.dataprotection/backup-create-storage-account-enable-protection/main.bicep#L97-L119

**Comment by sanjeevan11** [comment]
Yes, you need to include the policyParameters.backupDatasourceParametersList[].containersList inside the policyInfo to back up any configured containers in your storage account. This is because the default backup policy for storage accounts only backs up the root container.

The policyParameters.backupDatasourceParametersList[].containersList property is a list of container names that you want to back up. If you leave this property empty, the backup policy will not back up any containers.

**Comment by nilaydshah** [comment]
@ngbrown Updated the same template to support vaulted backup (which takes container list as input). Let me know if that is working well for you.
